docs/content/en/docs/reference/vsphere/vsphere-prereq.md

```diff
@@ -8,14 +8,18 @@ description: >
 
 To run EKS Anywhere, you will need:
 
+### Prepare Administrative machine
+Set up an Administrative machine as described in [Install EKS Anywhere ]({{< relref "../../getting-started/install/" >}}).
+
+### Prepare a VMware vSphere environment
+To prepare a VMware vSphere environment to run EKS Anywhere, you need the following:
 * A vSphere 7+ environment running vCenter
```

Comment:
Using `a vSphere 7+ environment` is misleading. When attempting to import the latest Ubuntu ovf specified on the artifacts page (`ubuntu-2004-kube-v1,21.5.ovf`) the process fails on vSphere 7+ running on Hosts ESXi 6.7 or lower. 

This is because the ovf actually specifies `<vssd:VirtualSystemType>vmx-15</vssd:VirtualSystemType>`, which refers to the [virtual hardware version](https://kb.vmware.com/s/article/1003746) of ESXi host and not vSphere itself. Perhaps there are API calls provided by vSphere 7+, but it should be made clear the ESXi hosts must be >= `ESXi 6.7 U2`. Additionally, it would be nice to have direct links to the Bottlerocket OVA for those who wish to manually download and import the OVA into vSphere.

Reply:
Alex, would you please commit your changes, and I will approve them from my side.